aggregations/db_tables/entity_added_accounts.py

```diff
@@ -0,0 +1,84 @@
+from ..sql_aggregations import SqlAggregations
+
+"""
+This code builds a table intended to assist with calculating statistics
+relating accounts ("added_accounts") to ecosystem entities ("entities")
+and is originally intended to be used for tracking new accounts added to each entity.
+"""
+
+
+class EntityAddedAccounts(SqlAggregations):
```

Comment:
We need to change this naming, it does not explain what's going on here 🙁 
`accounts_per_entity`
`accounts_owned_by_entities`
(I don't like them as well, but anyway)
Maybe you could suggest something else and we create a good name with collaborating 🙂 

I also think we need to add FK to `near_ecosystem_entities.slug`, it will explain one-to-many relation. Now it looks like many-to-many and it could confuse the users

Reply:
Agreed a better naming can be used here. `accounts_per_entity` is what i would have used, however it the figure represents only the number of accounts added on that particular day (not the running total). Would `accounts_added_per_entity` work for you? More than happy to go with either of your suggestions if you feel they describe the metric correctly.

Reply:
Let's go with `daily_accounts_added_per_entity`